CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes between versions
 
+## Not yet released
+
+* Hid internal command
```

Comment:
Hide*

Reply:
Could you submit a PR? If you have no time to finish it, @pyrech will finish it :)

Reply:
http://www.e-anglais.com/ressources/to_hide.php

Reply:
🙊

Reply:
they had been hiding internal command*